safe-global/web-core
# fix: do not allow threshold greater than owners

## What it solves

Resolves #758

## How this PR fixes it
Watches the threshold value on the Safe creation Owners policy step.
Adds a useEffect to guarantee that the threshold selected does not exceed the number of owners.

## How to test it
1. Add 3 owners when Safe creation.
2. Set threshold of 3
3. Delete one owner
4. The threshold adjusts automatically
5. Going to the next step you see the correct owners policy

## Analytics changes
None

## Screenshots
![threshold fixed](https://user-images.githubusercontent.com/32431609/193072350-34fb0fe1-fd5c-4d30-8eab-46301b9e79ba.gif)
